plugins/BEdita/API/src/Controller/Component/UploadComponent.php

```diff
@@ -80,6 +82,7 @@ public function upload($fileName, ?int $objectId = null): EntityInterface
         $private = filter_var($request->getQuery('private_url', false), FILTER_VALIDATE_BOOLEAN);
         $entity->set('private_url', $private);
         $data = $action(compact('entity', 'data'));
+        EventManager::instance()->dispatch(new Event('Thumbnails.update', $this, ['data' => $data]));
```

Comment:
I would add a try catch (logging the error) since `ImageThumbsHandler::updateThumbs()` throws an exception if there is a failure saving the image object.
I think that the thumbs creation must not block the upload action.